build/appveyor/before_build.bat

```diff
@@ -58,3 +58,22 @@ SET /p MSREVISION=<mscore\revision.h
 
 :: set ccache dir
 SET CCACHE_DIR=C:\ccache\cache
+
+IF "%BUILD_WIN_PORTABLE%" == "ON" (
+CD C:\MuseScore
+
+IF NOT EXIST portableappslauncher.zip (START " " /wait "C:\cygwin64\bin\wget.exe" --no-check-certificate "https://docs.google.com/uc?export=download&id=1hRaZTXY0ffm-TRLzu3a0_o77r3BEQcPf" -O portableappslauncher.zip )

```

Comment:
Here the archive should be retrieved from MuseScore server, not my Gmail account.
It is simply a zip of the installation folder of PortableApps.com Launcher.